build.gradle.kts

```diff
@@ -1,9 +1,16 @@
 plugins {
+
+    val kotlinVer = System.getenv("KOTLIN_VERSION_ENV")?.ifBlank { null } ?: libs.versions.kotlin.get()
```

Comment:
Why would we need to override the Kotlin version by setting an environment variable?

Reply:
To easily test with an eap version in the pipeline

Reply:
To easily test with an eap version in the pipeline